.github/workflows/release-alpha-v5.yml

```diff
@@ -25,12 +25,12 @@ jobs:
       - name: Setup
         uses: ./.github/actions/setup
 
-      - name: Version packages for canary
+      - name: Version packages for alpha-v5
         id: version-packages
-        run: npm run version:canary | tail -1 >> "$GITHUB_OUTPUT"
+        run: npm run version:alpha-v5 | tail -1 >> "$GITHUB_OUTPUT"
```

Comment:
Let's name the package scripts also `premajor` (the `alpha-v5` is an implementation detail and will change for v6, thus requiring us to change scripts in multiple places)